### ISSUE TITLE
Side pane is sometimes not restored in same width

### ISSUE THREAD

**Issue by xhc1** [quodlibet/quodlibet#3024]
pane is obviously smaller than was before or little bit bigger at new start ql

ql <= 4.1.0
Linux

**Comment by frestr** [comment]
Which pane (in which browser) do you mean?

**Comment by xhc1** [comment]
I am sure in: File System, Playlists, Internet Radio
maybe more, but not tried in my daily use

**Comment by ohel** [comment]
I can confirm with the newest master using the Internet Radio side pane. Steps to reproduce:
1. in Quod Libet, select any other browser than Internet Radio
2. select the Internet Radio browser
3. move your mouse cursor to where the pane ends
4. close Quod Libet
5. start Quod Libet. Your mouse cursor is not where the pane ends anymore.

It actually looks like so that on each startup, it makes the pane something like 70% of the width you last left it.

**Comment by CreamyCookie** [comment]
Is still the case in 4.2.1?

**Comment by ohel** [comment]
Yes, I just pulled the newest commit from master branch (4.3 development) and the problem is still there, using my repro steps from above.
